magda-web-client/src/Components/Dataset/Add/Pages/ReviewPage/CollapseBox.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, FunctionComponent } from "react";
+import "./CollapseBox.scss";
+import CollapseBoxContext, { DEFAULT_OPEN_STATUS } from "./CollapseBoxContext";
+import plusIcon from "assets/noun-plus.svg";
+import minusIcon from "assets/noun-minus.svg";
+import { withRouter, RouteComponentProps } from "react-router";
+import URI from "urijs";
+
+interface PropsType extends RouteComponentProps {
+    heading: string;
+    isOpen?: boolean;
+    stepNum: number;
+    className?: string;
+    isNotCollapsible?: boolean;
+}
+
+const CollapseBox: FunctionComponent<PropsType> = props => {
+    const isNotCollapsible =
+        typeof props?.isNotCollapsible === "undefined"
+            ? false
+            : props.isNotCollapsible;
+
+    const [isOpen, setIsOpen] = useState(
+        typeof props.isOpen === "boolean" ? props.isOpen : DEFAULT_OPEN_STATUS
+    );
+
+    const onToggleClick = () => {
+        setIsOpen(isOpen => (isOpen ? false : true));
+    };
+
+    return (
+        <div
+            className={`collapse-box row ${
+                props.className ? props.className : ""
+            }`}
+        >
+            <div className="col-sm-12">
+                <div
+                    className={
+                        isNotCollapsible
+                            ? "not-collapse-box-section-heading"
+                            : "collapse-box-section-heading"
+                    }
+                >
+                    {isNotCollapsible ? null : (
+                        <img
+                            src={isOpen ? minusIcon : plusIcon}
+                            onClick={onToggleClick}
+                        />
+                    )}
+
+                    <button
```

Comment:
Is it possible to make this a `<Link>` so you can hover over it to see where it goes / open in new tab / etc

Reply:
Changed to <Link>